packages/formation-react/src/components/Table/Table.stories.jsx

```diff
@@ -109,6 +109,19 @@ AlignLeft.args = {
   ],
 };
 
+// This story might not be useful
+export const alignRight = Template.bind({});
+alignRight.args = {
+  ...defaultArgs,
+  data: [
+    {
+      title: 'alignRight',
+      description:
+        'If you need to right align data in a column, in the fields object you can set alignRight to true for that field. This will right align the entire column which is generally used for numerical data in a table.',
+    },
+  ],
+};
```

Comment:
This doesn't actually demonstrate anything being aligned to the right:

![image](https://user-images.githubusercontent.com/2008881/98574513-df1d8c00-226c-11eb-9d9d-f913b5b2effb.png)


I think this would do a better job:

```diff
diff --git a/packages/formation-react/src/components/Table/Table.stories.jsx b/packages/formation-react/src/components/Table/Table.stories.jsx
index 88befaac7..67574c769 100644
--- a/packages/formation-react/src/components/Table/Table.stories.jsx
+++ b/packages/formation-react/src/components/Table/Table.stories.jsx
@@ -113,11 +113,13 @@ AlignLeft.args = {
 export const alignRight = Template.bind({});
 alignRight.args = {
   ...defaultArgs,
-  data: [
+  fields: [
+    defaultArgs.fields[0],
+    defaultArgs.fields[1],
     {
-      title: 'alignRight',
-      description:
-        'If you need to right align data in a column, in the fields object you can set alignRight to true for that field. This will right align the entire column which is generally used for numerical data in a table.',
+      label: 'Year',
+      value: 'year',
+      alignRight: true,
     },
   ],
 };
```

With `alignRight`:

![image](https://user-images.githubusercontent.com/2008881/98575166-b2b63f80-226d-11eb-84e0-56c203c86ac9.png)


Without `alignRight`:

![image](https://user-images.githubusercontent.com/2008881/98575209-c06bc500-226d-11eb-8652-ab43fdd18882.png)

Reply:
Yeah, I totally misunderstood the idea of a story LOL, just fixed this